src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -518,11 +518,11 @@ internal override string GetModulePath(PropertyScope scope)
         {
             if (PropertyScope.CurrentUser == scope)
             {
-                return ModuleIntrinsics.GetExpandedEnvironmentVariable("PSMODULEPATH", EnvironmentVariableTarget.User);
+                return ModuleIntrinsics.GetExpandedEnvironmentVariable("PSModulePath", EnvironmentVariableTarget.User);
```

Comment:
Whitespace changes - add `?w=1` to see the url to see diffs ignoring whitespace.